Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -314,11 +314,48 @@ extension StripeCardReaderService: CardReaderService {
             .eraseToAnyPublisher()
     }
 
+    public func retryActivePaymentIntent() -> AnyPublisher<PaymentIntent, Error> {
+        guard let activePaymentIntent = activePaymentIntent else {
+            return Fail(error: CardReaderServiceError.retryNotPossibleNoActivePayment)
+                .eraseToAnyPublisher()
+        }
+        switch activePaymentIntent.status {
+        case .requiresPaymentMethod:
+            return self.collectPaymentMethod(intent: activePaymentIntent)
```

Comment:
I think this `self.` is not needed:
```suggestion
            return collectPaymentMethod(intent: activePaymentIntent)
```

Reply:
Thanks 👍